itests/org.openhab.core.transform.tests/itest.bndrun

```diff
@@ -26,5 +26,5 @@ Fragment-Host: org.openhab.core.transform
 	org.openhab.core.storage.json;version='[2.5.0,2.5.1)',\
 	ch.qos.logback.core;version='[1.2.3,1.2.4)',\
 	slf4j.api;version='[1.7.25,1.7.26)',\
-	ch.qos.logback.classic;version='[1.2.3,1.2.4)',\
-	com.google.gson;version='[2.8.2,2.8.3)'
\ No newline at end of file
+	com.google.gson;version='[2.8.2,2.8.3)',\
+	slf4j.simple;version='[1.7.21,1.7.22)'
```

Comment:
It seems slf4j simple is now present on your runtime, too.
Need to check the dependency graph, why it is present.

Reply:
Ah, I added some time to be available for pure JUnit tests as an slf4j provider.
I Will exclude it for the test-index that is used by the integration tests.
I create a PR after some tests...

Reply:
I also noticed that. Will this be fixed wtih #652?

Reply:
If there is no other one that injects that bundle to the index it should be fixed.